### PR TITLE
[JENKINS-38114] Consolidated help for RunWrapper

### PR DESCRIPTION
**Pull request by jglick**
Downstream of https://github.com/jenkinsci/workflow-support-plugin/pull/13.

@reviewbybees

### CODE REVIEW

File: src/test/java/org/jenkinsci/plugins/workflow/WorkflowJobNonRestartingTest.java [hunk 1]
```diff
@@ -91,9 +93,15 @@ public void shellStep() throws Exception {
         // currentHeads[0] is FlowEndNode, whose parent is BlockEndNode for "node",
         // whose parent is BlockEndNode for body invocation, whose parent is AtomNode
         AtomNode atom = (AtomNode) e.getCurrentHeads().get(0).getParents().get(0).getParents().get(0).getParents().get(0);
-        LogActionImpl la = (LogActionImpl) atom.getAction(LogAction.class);
-        String text = FileUtils.readFileToString(la.getLogFile());
```
Comment (jglick) [inline src/test/java/org/jenkinsci/plugins/workflow/WorkflowJobNonRestartingTest.java]: Adapts to https://github.com/jenkinsci/workflow-support-plugin/pull/10.

File: src/main/resources/org/jenkinsci/plugins/workflow/cps/RunWrapperBinder/help.jelly [hunk 2]
```diff
@@ -1,14 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core">
-    <p>
-        The <code>currentBuild</code> variable may be used to refer to the currently running build.
-        It is an object similar to that documented for the return value of the <code>build</code> step (when <code>wait</code> is enabled).
```
Comment (jglick) [inline src/main/resources/org/jenkinsci/plugins/workflow/cps/RunWrapperBinder/help.jelly]: This quasi-link was the main motivation behind the issue. Now the actual content is visible here.

**Comment by abayer** [comment]
:bee:, since I believe that test is flaky.

**Comment by jglick** [comment]
> I believe that test is flaky

Hmm, no, it seems to fail in this branch, presumably due to a dependency update; will check why.

**Comment by jglick** [comment]
Caused by https://github.com/jenkinsci/workflow-api-plugin/pull/13.

**Comment by jglick** [comment]
or https://github.com/jenkinsci/workflow-support-plugin/pull/9

**Pull request merged**